identity-admin/webapp/utils/search-util.ts

```diff
@@ -1,38 +1,18 @@
 import { SortField } from '../interfaces';
-import * as queryString from 'querystring';
+import * as queryString from 'query-string';
 
 export function buildSearchParams(
   page: string | string[],
   status: string | string[],
   name: string | string[],
   email: string | string[],
   sortField?: string | string[],
-  sortDir?: string | string[]
+  sortDir?: string | string[],
 ): string {
-  let params = {};
-  if (page) {
-    params = { ...params, page: page };
-  }
-  if (status) {
-    params = { ...params, status: status };
-  }
-  if (name) {
-    params = { ...params, name: name };
-  }
-  if (email) {
-    params = { ...params, email: email };
-  }
-  if (
-    sortField &&
+  const sort = sortField &&
     typeof sortField === 'string' &&
-    Object.values(SortField).includes(sortField as SortField)
-  ) {
-    params = { ...params, sort: sortField };
-  }
-  if (sortDir && (sortDir === '1' || sortDir === '-1')) {
-    params = { ...params, sortDir: sortDir };
-  }
-  return queryString.stringify(params);
+    Object.values(SortField).includes(sortField as SortField) ? sortField : undefined;
```

Comment:
This was about ensuring that we're not passing invalid sort fields to the back end, since the query params are typed as `string | string[]` so could be anything but the backend will only accept one of several values. But now that I think about it, we should only end up in the situation where we've got an invalid sort field if the user is manually messing with the query string so it's not exactly a problem if they end up with some sort of error in that scenario. And we may even be able to handle it more gracefully in the integration layer. So I'm happy to remove this